apps/cowswap-frontend/src/common/updaters/orders/SpotPricesUpdater.ts

```diff
@@ -87,21 +87,32 @@ export function SpotPricesUpdater(): null {
         return
       }
 
-      const inputFraction = FractionUtils.fromPrice(inputPrice.price)
-      const outputFraction = FractionUtils.fromPrice(outputPrice.price)
-      const fraction = inputFraction.divide(outputFraction)
+      try {
```

Comment:
https://github.com/cowprotocol/cowswap/pull/5305/files#diff-a56f0408d1ca0aca16483d80cbe7622bdbc6e6ce9ea3adb0a56a4fecbd885990R90

@fairlighteth has already added this in one of his PRs

Reply:
Yes, I added it to his PR a few days ago when he was facing a crash.
Couldn't see it in the base PR so I added it again 😅 

Anyway, adding more error handling here in other places